WooCommerce/Classes/ViewRelated/Products/Edit Product/Composite Product Components/ComponentSettings.swift

```diff
@@ -0,0 +1,161 @@
+import SwiftUI
+import Kingfisher
+
+/// Renders the settings for a component in a composite product
+///
+struct ComponentSettings: View {
+
+    /// View model that directs the view content.
+    ///
+    let viewModel: ComponentSettingsViewModel
```

Comment:
Be aware of this ownership when the view model starts to get more dynamic

Reply:
Thanks for the reminder! I've made a note to check on this.